app/scripts/templates/sign_in.mustache

```diff
@@ -20,7 +20,7 @@
       <div class="error"></div>
       <div class="success"></div>
       {{#isAmoMigration}}
-        <div class="info nudge pad" id="amo-migration">{{#unsafeTranslate}}Looking for your Add-ons data? <a href="/signup">Sign up</a> for a Firefox Account with your old Add-ons account email address.{{/unsafeTranslate}}</div>
+        <div class="info nudge pad" id="amo-migration">{{#unsafeTranslate}}Looking for your Add-ons data? <a href="%(signUpWithSearchString)s">Sign up</a> for a Firefox Account with your old Add-ons account email address.{{/unsafeTranslate}}</div>
```

Comment:
@shane-tomlinson the PR should be good to go except of this ^ I keep getting `logger.js [sm]:73 String contains variables that are not escaped: Looking for your Add-ons data? <a href="%(signUpWithSearchString)s">Sign up</a> for a Firefox Account with your old Add-ons account email address.`